docs/tutorials/tutorial-custom-domain-with-istio.md

```diff
@@ -221,16 +232,19 @@ I want to install Istio with a default profile and SDS enabled. Furthermore I pa
     dns.gardener.cloud/dnsnames: "*.gsicdc.mydomain.io"
     dns.gardener.cloud/ttl: "120"
 ```
-With these annotations three things now happen automagically:
+
+With these annotations three things now happen automatically:
+
 1. The [External DNS Manager](https://github.com/gardener/external-dns-management/blob/master/README.md), provided to you as a service (`dns.gardener.cloud/class: garden`), picks up the request and creates the wildcard DNS entry `*.gsicdc.mydomain.io` with a time to live of 120sec at your DNS provider. My provider Cloud Flare is very very quick (as opposed to some other services). You should be able to verify the entry with `dig lovemygardener.gsicdc.mydomain.io` within seconds.
-2. The [Certificate Management](https://github.com/gardener/cert-management/blob/master/README.md) picks up the request as well and initates a DNS01 protocol exchange with Let's Encrypt; using the staging environment referred to with the issuer behind `mydomain-staging`.
-3. After aproximately 70sec (give and take) you will receive the wildcard certificate in the `wildcard-tls` secret in the namespace `istio-system`.
+1. The [Certificate Management](https://github.com/gardener/cert-management/blob/master/README.md) picks up the request as well and initiates a DNS01 protocol exchange with Let's Encrypt; using the staging environment referred to with the issuer behind `mydomain-staging`.
```

Comment:
Fix numbering
```suggestion
2. The [Certificate Management](https://github.com/gardener/cert-management/blob/master/README.md) picks up the request as well and initiates a DNS01 protocol exchange with Let's Encrypt; using the staging environment referred to with the issuer behind `mydomain-staging`.
```

Reply:
Numbering an ordered list as 

```
1. Item
1. Item
1. Item
```

will show up as:
1. Item
1. Item
1. Item

That way you do not have to keep track of which number each item is, and can focus on the actual content.

Reply:
Good to know!